components/toggle/toggle_constants.js

```diff
@@ -3,6 +3,8 @@ export const TOGGLE_SIZE_MODIFIERS = {
   md: '',
 };
 
+export const TOGGLE_CHECKED_VALUES = [false, true, 'mixed'];
```

Comment:
Is it possible to set it to "mixed" in storybook?

Reply:
Yeah, since the `checked` prop is used to set the initial state of the toggle, it should be added https://github.com/dialpad/dialtone-vue/pull/579/commits/2d4231724d703edaae69d747b8b4532ca3703dd1

Reply:
I simplified the approach here. Removed the `indeterminate` prop as this state can be managed with the `checked` value (`checked = "mixed"` means indeterminate toggle).